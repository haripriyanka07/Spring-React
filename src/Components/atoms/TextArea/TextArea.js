const TextArea = (props) => {
    return (
        <div>
            <textarea type={props.type} placeholder={props.placeholder} onChange={(e) => props.onChange(e.target.value)} value={props.value}></textarea>
        </div>
    )
}
export default TextArea;