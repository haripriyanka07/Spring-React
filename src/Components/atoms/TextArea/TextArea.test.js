import React from 'react';
import renderer from 'react-test-renderer';
import TextArea from './TextArea';

it('test the TextArea', () => {
    const renderedComponent = renderer.create(<TextArea value="this is textarea testing" />).toJSON();
    expect(renderedComponent).toMatchSnapshot();
})