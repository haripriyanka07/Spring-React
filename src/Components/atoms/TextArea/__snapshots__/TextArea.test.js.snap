// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the TextArea 1`] = `
<div>
  <textarea
    onChange={[Function]}
    value="this is textarea testing"
  />
</div>
`;
